.github/workflows/js-analysis.yml

```diff
@@ -0,0 +1,45 @@
+name: "JS Analysis tools"
+
+on:
+  push:
+    paths:
+      - '**.js'
+      - '**.scss'
+      - 'package.json'
+      - 'yarn.lock'
+      - 'jest.config.js'
+      - 'webpack.config.js'
+      - '.github/workflows/js-tests.yml'
```

Comment:
:thinking: why would we need to run that other workflow when changing this one and vice versa?